pkgs/development/python-modules/pdoc/default.nix

```
@@ -11,11 +11,12 @@
   astunparse,
   pytestCheckHook,
   hypothesis,
+  nix-update-script,
 }:
 
 buildPythonPackage rec {
   pname = "pdoc";
-  version = "14.7.0";
+  version = "15.0.0";
   disabled = pythonOlder "3.8";
```

```suggestion
  disabled = pythonOlder "3.9";
```
and we can remove astunparse from the dependencies.